resources/js/humhub.firebase.js

```diff
@@ -22,21 +33,55 @@ humhub.module('firebase', function (module, require, $) {
         }
     };
 
-    const afterServiceWorkerRegistration = function (registration) {
-        //console.log("After Service Worker Registration");
-        //console.log(registration);
+    /**
+     * Gets the content to display for notification permission status.
+     * @function
+     * @returns {string} The message corresponding to the current notification permission status.
+     */
+    const getNotificationPermissionContent = function () {
+        if (!("Notification" in window)) {
+            return module.text('status.not-supported');
+        }
+        switch (Notification.permission) {
+            case "granted":
+                return module.text('status.granted');
+            case "denied":
+                return module.text('status.denied');
+            default:
+                return module.text('status.default');
+        }
+    }
+
+    /**
+     * Adds information about push notification permissions to the UI.
+     * @function
+     * @param {string} permission - The notification permission status.
+     * @param {boolean} [isAfterRequest=false] - Indicates if this is after a permission request.
+     */
+    const addPushNotificationPermissionsInfo = function (permission, isAfterRequest = false) {
```

Comment:
@ArchBlood Seems the parameter `ìsAfterRequest` is now always `false` can you please remove? 

When `isAfterRequest` is false, the parameter `permission` also also unused. Can you please remove it too? :-)

Reply:
Done in https://github.com/humhub/fcm-push/pull/44/commits/964babe6d04b3742c3e7909bc9fd4ce1e4c08344, although I'd like mention that the listener will always return true now and the message channel will close before it is received in some cases. Also worth mentioning that it will also throw new console errors when it comes to the endpoint for `/unsubscribe` not being implemented, so I'm not sure what more I'd be able to do in these cases. 🤔